classification/classifier_main.py

```diff
@@ -10,15 +10,13 @@ class Classifier:
 
     def __init__(
             self,
-            normal_text_contents,
-            title_text_contents,
-            media_contents,
-            embedded_contents,
+            contents_object,
             max_pages):
-        self.normal_text_count = len(normal_text_contents)
-        self.title_text_count = len(title_text_contents)
-        self.media_count = len(media_contents)
-        self.embedded_content_count = len(embedded_contents)
+        self.normal_text_count = len(contents_object.normal_text)
+        self.title_text_count = len(contents_object.title_text)
+        self.media_count = len(contents_object.media)
+        self.embedded_content_count = len(contents_object.embedded_content)
+        self.quoted_content_count = len(contents_object.quoted_content)
```

Comment:
All these can be methods in PreprocessedContents.
Create the methods directly/in-place and avoid creating lots of variable.

Reply:
updated accordingly.